colorcet/sineramp.py

```diff
@@ -80,7 +80,7 @@
 visual performance of the viewer.
 
 The Default Amplitude:
-This is set at 12.5 so that from peak to trough we have a local feature of
+This is set at 12.5 so that from peak to through we have a local feature of
```

Comment:
```suggestion
This is set at 12.5 so that from peak to trough we have a local feature of
```

(The original is correct.)